roadmaps/2022.md

```diff
@@ -91,33 +91,73 @@ The core team has two main goals for this quarter: to improve the product and al
+- [X] Work with the devrel team to promote the contributions we need on the tokenizer side -> done during Hacktoberfest, so Q4 2022.
+- [X] Keep improving indexation speed/crash
+  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement -> We are working on creating use-case benchmarks in Q4 2022.
+  - [X] by keep investigating and trying new technical solutions on our side -> we made a lot of changes described in [this issue](https://github.com/meilisearch/meilisearch/issues/2132).
+
+## Q4 2022
+
+This quarter is the last one we have to prepare v1 in good condition. This way, we have prioritized tasks that would help the team deliver Meilisearch v1: a stable and impactful version of Meilisearch.
+It also means we will de-prioritize implementations that could impact in a bad way the critical features of Meilisearch, like improving the indexing speed in a way that would risk making the indexing fail.
+
+Q4 is also the quarter where the Hacktoberfest is happening and this will lead us to spend time on it. We created a lot if issues during Q3, especially regarding the language support, and we hope attract the community and make them contribute in October.
+
+### v1 work
+
+- [ ] Be production ready for v1
+  - [ ] Release the missing production-ready features: index swap, cancel/delete of tasks, and other missing features noticed by the product team.
+  - [ ] We know Meilisearch can have bugs/crashes impacting the stability of the search engine in production. In the best world, the RAM consumption should be appropriate when indexing and searching. It means our priority during this quarter is to identify the different weaknesses of Meilisearch and make decision about them: being transparent with the users is the most important part to achieve if the problem cannot be solved by any implementation before v1.
+  - [ ] Define more precisely what kind of distribution we support. Transparency is the key to this point. Then, we need to ensure we have tests/CIs appropriate to them.
+  For example, what about the Debian 10 distribution? And what about Windows and the prioritization we should give to it? Indeed, all the features of Meilisearch do not work with Windows so far, like dumps. It's important to provide transparency about it.
+- [ ] Define our versioning policy process post v1
+- [ ] Define our beta/deprecated process post v1
+- [ ] Create some benchmarks of concrete use cases. We need to define the real objectives of the project, how to ensure to make it useful at the end. Create a Product Squad for it and manage it as a new feature of Meilisearch could be a good start.
+
+### Team
+
+- [ ] Onboard the new senior software engineer: at the beginning of November, we welcome a new software engineer to the team.
+- [ ] This is the end of the internship of one of our teammates. Following a promotion, we want to integrate him in the sprint as any developer of the team.
```

Comment:
My personal opinion on this is: I don't like to see hiring on the roadmap. It happens every month on most of the teams, and it should be seen as something that is usual. I know it takes time, but the goal of a Roadmap is not to justify the time spent on a quarter. It's mostly to show the next priorities. Mainly what could impact the users or the product.

Reply:
Hum ok, I understand. However, the roadmap should also justify why why it could be lighter than usual, and the onboardings are a justification of it. Is it ok if I mention it at the beginning of the roadmap instead of "being part of it" like any other tasks?

Reply:
> Mainly what could impact the users or the product.

From my understanding, it is more of an internal roadmap for the core team on upcoming tasks that need to be done to improve the team's daily life and ensure its success.

Reply:
I think you don't have to justify anything. It's just about getting information to others about the direction you're taking.

Reply:
Yes, it's indeed not the product roadmap, but the team objective, even if it's tightly related to the product roadmap
Maybe the current format we have (i.e having 2 roadmap) is not relevant anymore, and everything should be put here?

Reply:
As discussed in person, I'll try to find a way to better document what is required for the roadmaps. No need to update this one today.